### ISSUE TITLE
Time selection uses both 12 hour and 24 hour format.

### ISSUE THREAD

**Issue by TwoRice** [indrimuska/angular-moment-picker#206]
When selecting a time, when choosing the hours the time is in 24 hr format and then when selecting the minutes it converts to 12 hr format. This is really annoying and confusing; the first time I used it I thought something had went wrong, because I had selected 1800, but I thought I was then selecting minutes for 0600. The time format should be consistent throughout the selection process.

**Comment by jlubeck** [comment]
Same issue here. Were you able to solve it?

**Comment by TwoRice** [comment]
No, I'm not currently using the component, just noticed it when I was looking for time pickers. I reckon its a design flaw rather than a bug.

**Comment by pmaiya** [comment]
I am having the same issue. Any help here would be great.

**Comment by pmaiya** [comment]
I found a solution from here : https://github.com/indrimuska/angular-moment-picker/issues/2
Add a config to your module like this: 
angular
    .module('myApp', ['moment-picker'])
    .config(['momentPickerProvider', function (momentPickerProvider) {
        momentPickerProvider.options({
            minutesFormat: 'HH:mm'
        });
    }]);

**Comment by Alka2209** [comment]
Hi. I am using Amazing timepicker in angular4. here, i stuck in one point. where i want to give the time in 12hrs format. as by default amazing timepicker is wrking in 24hrs format. i want if users select 4:30pm then it shows 4:30pm in watch not 16:30.

please help me out asap.

**Comment by Alka2209** [comment]
> I found a solution from here : #2
> Add a config to your module like this:
> `angular .module('myApp', ['moment-picker']) .config(['momentPickerProvider', function (momentPickerProvider) { momentPickerProvider.options({ minutesFormat: 'HH:mm' }); }]);`

hey can u help me?
Hi. I am using Amazing timepicker in angular4. here, i stuck in one point. where i want to give the time in 12hrs format. as by default amazing timepicker is wrking in 24hrs format. i want if users select 4:30pm then it shows 4:30pm in watch not 16:30.

please help me out asap

**Comment by kshitijak-iprogrammer** [comment]
> > I found a solution from here : #2
> > Add a config to your module like this:
> > `angular .module('myApp', ['moment-picker']) .config(['momentPickerProvider', function (momentPickerProvider) { momentPickerProvider.options({ minutesFormat: 'HH:mm' }); }]);`
> 
> hey can u help me?
> Hi. I am using Amazing timepicker in angular4. here, i stuck in one point. where i want to give the time in 12hrs format. as by default amazing timepicker is wrking in 24hrs format. i want if users select 4:30pm then it shows 4:30pm in watch not 16:30.
> 
> please help me out asap

did u got any solution??
